src/Service/DigitalPostHelper.php

```diff
@@ -2,19 +2,24 @@
 
 namespace App\Service;
 
+use App\Entity\DigitalPost as DigitalPostEntity;
```

Comment:
Consider adding `use ItkDev\Serviceplatformen\DigitalPost\DigitalPost as DigitalPostBase` instead for the base class. Using `DigitalPostEntity` feels weird.

Reply:
Used `DigitalPostBase` rather than `DigitalPostEntity`.